x-pack/plugins/upgrade_assistant/public/application/components/overview/deprecation_logging_toggle.tsx

```diff
@@ -7,25 +7,39 @@
 
 import React, { useEffect, useState } from 'react';
 
-import { EuiLoadingSpinner, EuiSwitch } from '@elastic/eui';
+import { EuiSwitch, EuiLink, EuiFormRow, EuiButton, EuiSpacer, EuiCallOut } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
 
+import { SectionLoading } from '../../../shared_imports';
 import { useAppContext } from '../../app_context';
 import { ResponseError } from '../../lib/api';
 
 const i18nTexts = {
-  toggleErrorLabel: i18n.translate(
-    'xpack.upgradeAssistant.overviewTab.steps.deprecationLogsStep.enableDeprecationLoggingToggleSwitch.errorLabel',
+  fetchErrorMessage: i18n.translate(
+    'xpack.upgradeAssistant.overviewTab.steps.deprecationLogsStep.enableDeprecationLoggingToggleSwitch.fetchErrorMessage',
     {
       defaultMessage: 'Could not load logging state',
     }
   ),
+  reloadButtonLabel: i18n.translate(
+    'xpack.upgradeAssistant.overviewTab.steps.deprecationLogsStep.enableDeprecationLoggingToggleSwitch.reloadButtonLabel',
+    {
+      defaultMessage: 'Reload',
```

Comment:
Can we add some more information so the user understands why they'd want to bother reloading? For example, we could change the button text to "Reload to configure deprecation logging".